src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -298,27 +321,36 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
     }
   }
 
-  private showLoadingMessage(): void {
+  private showLoadingMessage(isCertificateRenewal = false): void {
     if (this.currentStep !== E2EIHandlerStep.ENROLL) {
       return;
     }
 
     const {modalOptions, modalType} = getModalOptions({
       type: ModalType.LOADING,
       hideClose: true,
+      extraParams: {
+        isRenewal: isCertificateRenewal,
+      },
     });
     PrimaryModal.show(modalType, modalOptions);
   }
 
-  private showSuccessMessage(): void {
+  private showSuccessMessage(isCertificateRenewal = false): void {
     if (this.currentStep !== E2EIHandlerStep.SUCCESS) {
       return;
     }
+
     const {modalOptions, modalType} = getModalOptions({
       type: ModalType.SUCCESS,
-      primaryActionFn: () => this.emit('enrollmentSuccessful'),
```

Comment:
why moving this? I have put it there so that the app doesn't start showing while the popup is visible. 
Is that a problem?

Reply:
I didn't remove this intentionally,  got removed while resolving conflicts I suspect. I was thinking about whether always emitting enrollmentSuccessful from showSuccessMessage can be an issue. Its fine as thats the only type of event we emit for showSuccessMessage modal.

I still see this.emit('enrollmentSuccessful'); at line number 312 in my PR, does that need to be removed then?

```
 this.showSuccessMessage(isCertificateRenewal);

 this.emit('enrollmentSuccessful');
```